docs/conditions.md

```diff
@@ -57,10 +58,31 @@ only start with alpha characters and `_`. For example, `fooIs-Bar_` is a valid
 parameter name, `barIsBa$` or `0banana` are not.
  
 Each declared parameter has a type field, assumed to be string if not provided by the user. 
-The other possible type is array — useful, checking a pushed branch name doesn't match any of 
+The other possible type is array — useful, for instance, checking that a pushed branch name doesn't match any of 
 multiple protected branch names. When the actual parameter value is supplied, its parsed type 
 is validated against the type field.
 
+### Resources
+
+Conditions can declare input [`PipelineResources`](resources.md)  via the `resources` field to 
+provide the Condition container step with data or context that is needed to perform the check.
+
+Input resources, like source code (git), are dumped at path
+`/workspace/resource_name` within a mounted
```

Comment:
Yeah, the thing is since things like `targetPath` are part of `TaskResource`, they are in the tasks doc. Now with conditions, there is some duplication. I'll add a section in the resource doc on how to use resources in conditions and tasks. That section will link to both the Task and Condition resource docs and contain the common bits e.g. targetPaths, the path variable substitution.

I'm thinking of doing this in a follow-up PR.